plugins/inputs/cpu/cpu.go

```diff
@@ -125,10 +147,13 @@ func activeCPUTime(t cpuUtil.TimesStat) float64 {
 
 func init() {
 	inputs.Add("cpu", func() telegraf.Input {
-		return &CPUStats{
+		cpuStats := &CPUStats{
 			PerCPU:   true,
 			TotalCPU: true,
 			ps:       system.NewSystemPS(),
 		}
+		cpuStats.initCPUInfo()
+
+		return cpuStats
```

Comment:
Please keep this as it was after changing `initCPUInfo` to `Init` as suggested above.